solaris/solaris10/README.md

```diff
@@ -8,20 +8,20 @@ In this repository, you can find the necessary tools to build a Wazuh package fo
 To build a Wazuh package you need to install the following tools:
   - `git`: `pkgutil -i git`.
 
-## Building RPM packages
+## Building Solaris11 packages
 
 To build a Solaris 10 package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository and generate a `.pkg` package.
 
-1. Download this repository and go to the rpm directory:
+1. Download this repository and go to the `solaris/solaris10` directory:
     ```bash
-    $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/solaris10
+    $ git clone https://github.com/wazuh/wazuh-packages && cd wazuh-packages/solaris/solaris10
     ```
 
-2. Execute the `generate_wazuh_packages.sh` script to build the package. There are multiple parameters to select which package is going to be built, its architecture, etc. Here you can see all the different parameters:
+2. Execute the `generate_wazuh_packages.sh` script to build the package. Here you can see all the different parameters:
     ```shellsession
     # ./generate_wazuh_packages.sh -h
 
-    Usage: ./generate_wazuh_rpm.sh [OPTIONS]
+    Usage: ./generate_wazuh_packages.sh [OPTIONS]
```

Comment:
Please change the `Build deb packages.` part.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893